documentation/cloud-manager-guide/secure-data-connectors/add-data-connector.md

```diff
@@ -0,0 +1,72 @@
+---
+keywords:
+title: Add a Data Connector
+description: Add a data connector
+---
+# Add a Data Connector
+
+This guide outlines the steps to add a new secure data connector in Environment Operations Center.
+
+[!info] The steps outlined in this guide assume you have already configured the OPA client to establish the WebSocket tunnel between your on-prem environment and Env Ops Center. For details on how to configure the OPA client, see the [configure an OPA client](...) guide. **(update link when available)**
+
+## Getting started
+
+Adding a new data connector in Env Ops Center is broken out into two steps: Creating the data connector by providing its basic information and registering the data connector.
+
+To add a new data connector, select **Add New Connector** from the *Secure Data Connectors* home screen.
+
+![image description](images/new-connector.png)
+
+## Add data connector info
+
+In the *New Data Connector* dialog, enter the required data connector info in the provided name and a description fields.
```

Comment:
Change "info" to "information" or copy "Data Connector info" directly from the UI and make it clear we're referencing the in-UI title here. Are these names and descriptions they make up? Looks like only Name is required, so carful wording on that. What is shown now looks like both are required and that is not accurate.